test/t/ngx_mruby.rb

```diff
@@ -585,6 +585,22 @@ def _run
   t.assert_equal "", client_h.run_again.chomp
 end
 
+t.assert('ngx_mruby - validate client certificate without validation error') do
+  path = ENV['NGINX_INSTALL_DIR'] + '/html'
+  res = OpenSSLTestClient.new
+        .run("openssl s_client -servername localhost -cert #{path}/client.crt -key #{path}/client.key -CAfile #{path}/localhost.crt -connect localhost:58072")
+        .pipe("grep 'Verify return code' | awk -F'[:]' '{print $2}'")
+  t.assert_equal " 0 (ok)\n", res
+end
+
+t.assert('ngx_mruby - invalid client certificate') do
+  path = ENV['NGINX_INSTALL_DIR'] + '/html'
+  res = OpenSSLTestClient.new
+        .run("openssl s_client -servername localhost -cert #{path}/client.crt -key #{path}/client.key -CAfile #{path}/localhost.crt -connect localhost:58073")
+        .pipe("grep 'Verify return code' | awk -F'[:]' '{print $2}'")
+  t.assert_equal " 18 (self signed certificate)\n", res
```

Comment:
It's very hard to bypass some validations for openssl 1.1.

I tried to add our test `localhost.crt` and `dummy.crt` to trusted certificates like to follow, but I have got still same failure "self signed certificate". If you have an idea how we make it fail actual client certificate verification, I'm all ears.

```
cp build/nginx/html/dummy.crt /usr/local/share/ca-certificates/dummy.crt
cp build/nginx/html/dummy.crt /usr/local/share/ca-certificates/localhost.crt
update-ca-certificates
```